ocrd/ocrd/cli/workspace.py

```diff
@@ -159,6 +167,54 @@ def workspace_add_file(ctx, file_grp, file_id, mimetype, page_id, force, local_f
     )
     workspace.save_mets()
 
+# ----------------------------------------------------------------------
+# ocrd workspace add-group
+# ----------------------------------------------------------------------
+
+@workspace_cli.command('add-group')
+@click.option('-m', '--mimetype', help="Media type of the file", required=True)
+@click.option('-f', '--force', help="Continue even if mets:file same ID already exists", default=False, is_flag=True)
+@click.argument('file_grp_dir', type=click.Path(dir_okay=True, file_okay=False, readable=True, resolve_path=True), required=True)
+@pass_workspace
+def workspace_add_file_group(ctx, mimetype, force, file_grp_dir):
+    """
+    Add a directory of files as a fileGrp to METS.
+
+    Convention:
+        FILEGRP_DIR must reside within the workspacee
+        fileGrp ID = last folder name component
+        page ID = filename sans extension
+        file ID = sanitized concatenation of fileGrp ID and page ID
+    """
+    workspace = Workspace(ctx.resolver, directory=ctx.directory, mets_basename=ctx.mets_basename, automatic_backup=ctx.automatic_backup)
+
+    file_grp_dir = Path(file_grp_dir).resolve()
+
+    ws_dir = Path(ctx.directory).resolve()
+    print(file_grp_dir)
```

Comment:
no logging? (Also recommend logging file ID and pageId of every file to be added in the loop.)

Reply:
That was just a debug stmt. Logging is indeed missing, I'll add some.

Reply:
logging added.